src/EventHandler/ConfigurableEventHandlerBase.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rules\EventHandler\ConfigurableEventHandlerBase.
+ */
+
+namespace Drupal\rules\EventHandler;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\rules\Core\RulesConfigurableEventHandlerInterface;
+use Drupal\rules\Core\RulesDefaultEventHandler;
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Base class for event handler.
+ */
+abstract class ConfigurableEventHandlerBase extends RulesDefaultEventHandler implements RulesConfigurableEventHandlerInterface {
+
+  /**
+   * The event configuration.
+   *
+   * @var array
+   */
+  protected $configuration = array();
+
+  /**
+   * @inheritdoc
+   */
+  public static function determineQualifiedEvents(Event $event, $event_name, array &$event_definition) {
+    // Nothing to do by default.
+  }
+
+  /**
+   * @inheritdoc
+   */
+  public function summary() {
+    // Nothing to do by default.
```

Comment:
Method or comment? If I remove the method I have to implement it in the ConfigurableEventHandlerEntityBundle. What to put in there?